gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -211,7 +209,13 @@ private void observeMultipleChoiceClicks(MultipleChoiceFieldViewModel viewModel)
 
   private void onShowDialog(
       Field field, Optional<Response> currentResponse, Consumer<Optional<Response>> consumer) {
-    Cardinality cardinality = field.getMultipleChoice().getCardinality();
+
+    MultipleChoice multipleChoice = field.getMultipleChoice();
+    if (multipleChoice == null){
```

Comment:
Can we use Guava's [checkNotNull](https://guava.dev/releases/19.0/api/docs/com/google/common/base/Preconditions.html#checkNotNull(T)) instead? It can be statically imported from `Preconditions`.

Reply:
Great suggestion. Replaced `if (x == null){ throw NPE(message); }` with `Preconditions.checkNotNull(x, message)` here and throughout.